build.gradle

```diff
@@ -203,7 +210,7 @@ dependencies {
   testCompile group: 'org.junit.vintage', name: 'junit-vintage-engine', version: versions.junit
   testCompile group: 'org.mockito', name: 'mockito-junit-jupiter', version: '2.23.4'
   testCompile group: 'org.assertj', name: 'assertj-core', version: '3.11.1'
-  testCompile group: 'org.springframework.cloud', name: 'spring-cloud-contract-wiremock', version: '2.0.2.RELEASE'
+  testCompile group: 'org.springframework.cloud', name: 'spring-cloud-contract-wiremock', version: '2.1.0.RELEASE'
```

Comment:
worth checking if the integration tests works fine as it was failing previously but yeah pipeline will check it anyways

Reply:
i'm afraid with our current integration tests (wiremock setup specifically 🙄) this will have to go away, again